package.json

```diff
@@ -41,6 +41,7 @@
     "@floating-ui/react": "^0.26.17",
```

Comment:
Can we remove the following dependencies? Or would they still be required to use the new components library? If so, would we need to document the requirements?
`@floating-ui/react`
`@vscode/webview-ui-toolkit`
`antd`
`react-markdown`
`remark-gfm`
`throttle-debounce`
`vscode-messenger-webview`

On `primeflex`: I was able to remove it and the one remaining import. And didn't see anything suspicious at runtime. But at the same time only tested with VS Code, not the web variant. Please remove if you believe we don't need it anymore.

Reply:
Removed.